### DISCUSSION TITLE
A problem of including library

### DISCUSSION THREAD

**Discussion by huanmoon** [clangd/clangd#1802] [Q&A]
I'm using clangd with coc-clangd in neovim and I am writing a c++ file with library ncurses.h
the functions in this library are very similar to curses.h so clangd include curses.h automaticlly, but I just want to use ncurses.h not curses.h
I've post an issue in coc-clangd and someone told me that it is an issue of clangd, not an issue of coc-clangd(https://github.com/clangd/coc-clangd/issues/633)
Can someone tell me how to fix it?
Thanks in advance! =)

**Comment by HighCommander4** [comment]
Filed in duplicate at #1803
